tests/js/server/aql/aql-functions-date.js

```diff
@@ -2457,6 +2457,37 @@ function ahuacatlDateFunctionsTestSuite () {
 	  assertEqual([ "2020-02-29T23:00:00.000Z" ], 
 		getQueryResults("RETURN DATE_LOCALTOUTC(DATE_ADD(DATE_UTCTOLOCAL('2020-01-31T23:00:00.000Z', 'Europe/Berlin'), 1, 'months'), 'Europe/Berlin')"));	  
     },
+	
+	// //////////////////////////////////////////////////////////////////////////////
+    // / @brief test date_timezone
+    // //////////////////////////////////////////////////////////////////////////////
+
+    testDateTimeZone: function () {
+	  var systemtz = null;
+	  var res = getQueryResults("RETURN DATE_TIMEZONE()", {})[0];
+	  
+	  try {
+	    systemtz = Intl.DateTimeFormat().resolvedOptions().timeZone;
+	  } catch {
+	  }
+		
+	  if (systemtz) {
+	    assertEqual(systemtz, res);
+	  } else {
+		assertTrue(res != null);
```

Comment:
Do I understand correctly that when there is no systemtz, the DATE_TIMEZONE function should still return non-null? I would assume that it will return null then.
No matter what, instead of `assertTrue(res != null);` you can use `assertNotNull(res);`, which is more explicit and intuitive. 
(apart from that, using `res != null` could make our jslint scan for the JavaScript files fail, because it may demand a `res !== null` here. Not sure exactly, but it may happen at least for other comparisons. But it would be best to avoid this and use `assertNotNull` (or `assertNull`) instead.
By the way, using either `assertNotNull` or `assertNull` requires adding these "global" helper functions to the list of global function names at the very beginning of the test case file (line 2).